src/app/_index.js

```diff
@@ -101,6 +102,7 @@ ipcRenderer.on('export-file-from-menu', (event, { type }) => {
     logger,
     exportSaveDialog,
     MPQ,
+    rm,
```

Comment:
wouldn't we want to use the rmRF here instead?

Reply:
Yup